lib/l10n/app_de.arb

```diff
@@ -46,6 +46,7 @@
   "invalidUrlErrorText": "Dieser Link scheint ungültig zu sein.",
   "duplicateUrlErrorText": "Dieser Link existiert schon",
   "or": "oder",
+  "activityFeed": "Aktivitäts-Feed",
```

Comment:
This seems to be a dupe on line 20 as well @kevmo314 should the variables be renamed or should be consolidate down to one?

Reply:
Resolved in #1289